### DISCUSSION TITLE
Date validation before or after the Current date

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#18819] [Feature Requests]
### Summary

In the case we want the date to be before (or after) today, it would be nice to have a `Today` value when establishing the validation.

### Basic Example

Date Less than Today

### Motivation

In a Product list, I have a ended date (while current product have this field to null).
The date _has_ to be in the past

### Detailed Design

On creating the Validation rule,  the dropdown could show the ability to set the right part to `Today`

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

Presently we have a calendar for fixed dates.
It implies to show for instance a "Today" or "Current date" at the bottom.

### Alternatives

-

### Adoption Strategy

-

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
